src/legacy/constants/routes.json

```diff
@@ -25,5 +25,6 @@
   "CHAT": "/chat",
   "PORTFOLIO_DETAIL": "/portfolio_detail",
   "WELCOME_PAGE": "/welcome-page",
-  "LOGIN": "/login"
+  "LOGIN": "/login",
+  "SEARCH_RESULT": "/search_result"
```

Comment:
Please move to new routes

Reply:
I think we are using legacy route now. And legacy/Routes.tsx, it is using routes.json in legacy folder. So it should be changed for routing to test. Or we need to use new routes.json file in legacy/Routes.tsx.

Reply:
Yes, we have used the legacy route before but I think it is not right because we are working on the new application/code so I think we should move to the new routes.